tensorflow/python/util/compat.py

```diff
@@ -35,9 +35,13 @@
 
 import numpy as _np
 import six as _six
+try:
+  from collections.abc import Iterable as _Iterable
+except ImportError:
+  from collections import Iterable as _Iterable
 
 from tensorflow.python.util.tf_export import tf_export
-
+from tensorflow.python.framework import ops
```

Comment:
Looks like this is causing a circular dependency. Since there's only one user of the new function, I'd move it to `readers.py`.

Reply:
IIUC the only caller is in `readers.py`, so the simplest solution is to move the function to that module, which breaks the circular dependency. We can figure out something more sophisticated if the conversion function turns out to be needed in another place.